src/lib/components/Modal.svelte

```diff
@@ -27,4 +42,29 @@
       background-color: $tertiary
       overflow: auto
 
+      .title 
+        font-weight: 700
+        font-size: 2rem
+        line-height: 36px
```

Comment:
Could these values be design tokens ?

Reply:
Yes It could, but a bit late for the project ? Or it could be an entire subject for a "future" PR ?